plugins/vlc-video/vlc-video-source.c

```diff
@@ -498,6 +514,8 @@ static void vlcs_update(void *data, obs_data_t *settings)
 	size_t count;
 	int network_caching;
 	int track_index;
+	int subtitle_index;
+	int subtitle_enable;
```

Comment:
why an int and not a bool ?

Reply:
Thank you for this and your work on the audio track update, obviously helped point me in the right places.